src/components/Store.react.js

```diff
@@ -126,9 +123,9 @@ export default class Store extends React.Component {
         super(props);
 
         if (props.storage_type === 'local') {
-            this._backstore = _localStore;
+            this._backstore = new WebStore(window.localStorage);
```

Comment:
The error message we get when trying to access `localStorage` is fairly informative / helpful:

```
Uncaught DOMException: Failed to read the 'localStorage' property from 'Window': The document is sandboxed and lacks the 'allow-same-origin' flag.
```

I don't think it needs to be improved to make sense to an app developer and as such can be exposed as-is.